r/src/RecordBatch.cpp

```diff
@@ -95,20 +104,20 @@ int RecordBatch__to_file(const std::shared_ptr<arrow::RecordBatch>& batch,
   return offset;
 }
 
-// [[Rcpp::export]]
-RawVector RecordBatch__to_stream(const std::shared_ptr<arrow::RecordBatch>& batch) {
+int64_t RecordBatch_size(const std::shared_ptr<arrow::RecordBatch>& batch) {
```

Comment:
Hm, "stream_size"?

Reply:
That makes sense. I’ve switched to the other side of the problem. I’ll keep this in mind and propose some changes.